### PR TITLE
CTP-1685 add unfreeze function to lifecycle block

### PR DESCRIPTION
**Pull request by aydevworks**
Deployed on 44-clc.

### CODE REVIEW

**Comment by aspark21** [comment]
Code reviewed & tested.

When unfreezing, this gives 1 day before the course is frozen again. 
I don't see anywhere where this is defined though?

**Comment by aydevworks** [comment]
> Code reviewed & tested.
> 
> When unfreezing, this gives 1 day before the course is frozen again. I don't see anywhere where this is defined though?

Yes, I forgot to disable the auto context freezing for the course after unfreeze, so it will be frozen again in the next day when the scheduled task runs.

**Comment by aspark21** [comment]
I think 1 day was fine but maybe we can have a pop up warning them it will refreeze unless they go and change settings in the block

**Comment by aydevworks** [comment]
> I think 1 day was fine but maybe we can have a pop up warning them it will refreeze unless they go and change settings in the block

Updated and deployed to 44-clc.

File: templates/unfreeze_button.mustache [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<button type="button"
+        class="btn btn-primary w-100"
+        data-confirmation="modal"
+        data-confirmation-title-str='["label:unfreezebutton", "block_lifecycle"]'
+        data-confirmation-content-str='["confirm:unfreeze_button", "block_lifecycle", {"contextname": "{{coursename}}"}]'
+        data-confirmation-yes-button-str='["confirm"]'
+        data-confirmation-destination="{{url}}"
+>
+    Enable editing
+    <span class="sr-only"> for course {{coursename}}</span>
```
Comment (aspark21) [inline templates/unfreeze_button.mustache:46]: That looks like hardcoded text that should be in language strings

Comment (aspark21) [reply templates/unfreeze_button.mustache:46]: "for course" is still hardcoded in english

**Comment by aspark21** [comment]
Tested and code reviewed. Have made some minor changes to the language strings for clarity. 

Once the hardcoded words are moved to language strings, this can be merged

**Review by aspark21 — APPROVED** [review]

**Pull request merged**